### PR TITLE
NEP-12: NewORG Standard (NRC-12)

### PR DESCRIPTION
**Pull request by i29**
## Checklist:

- [x] I have read the **NEP Guideline** document.
- [x] I have checked my NEP Draft is ready.

### CODE REVIEW

**Comment by arisac** [comment]
Review Checklist for NEP Editors

`this checklist is for new NEP Draft`

**Check pr NEPs repo**
- [ ] new NEP is placed in /NEPS/nep-title.md
- [ ] assets are placed in /assets/NEP-N/asset-name.ext
- [ ] no modifications to other files

**Check NEP formats**
- [ ] header
- [ ] all contents in English

Ask the applicant to make modifications to meet criterions above.

**Assign NEP Number**
- [ ] use pr number if no related issue had opened before
- [ ] change Title and NEP number in document
- [ ] rename nep-title.md to nep-number.md
- [ ] rename assets directory NEP-N to nep-number

**Arrange NEP Board Review**
tbd

File: NEPS/nep-NewORG-Standard.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
+# NewORG  Standard (NRC-X)
+
+| Item | Description |
+|:-|:-|
+| NEP | to be assigned |
+| Title | New paradigm of organization |
+| Author |  [Evan Liu](mailto:evanliuchina@gmail.com), [Xia Wu](https://github.com/xiawu), [Lee Willson](https://github.com/leewillson), [Zhou Xiqiao](https://github.com/zhouxiqiao), [Yong Liu](mailto:liuyong5653@163.com), [Qu Jianwei](https://github.com/i29) |
+| Discussions to | https://github.com/NewORG-Standard-Group/NEPs/issues/1 |
+| Status | WIP |
```
Comment (arisac) [inline NEPS/nep-NewORG-Standard.md:9]: please change

File: NEPS/nep-NewORG-Standard.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+| Type | NewOrg Template |
```
Comment (arisac) [inline NEPS/nep-NewORG-Standard.md:10]: are you suggesting we create a new type `NewOrg Template`?

if not the case, just use Standard Track

**Comment by arisac** [comment]
@i29 please check [x] `Allow edits from maintainers` for this pr.

**Comment by arisac** [comment]
Please use 12 as your NEP number

**Comment by i29** [comment]
@arisac
 is right ?

**Comment by arisac** [comment]
@i29  :thumbsup: your document is placed in `/NEPS/nep-12.md`

but 

:point_right: assets should be placed in `/assets/nep-12/name.ext`

:point_right: links to assets in docs should use `../assets/nep-12/name.ext`

**Comment by i29** [comment]
@arisac fixed

File: NEPS/nep-12.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
+# NEP 12: NewORG Standard (NRC-X)
```
Comment (evanliuchina) [inline NEPS/nep-12.md:1]: should change NRC-X to NRC-12 also?

File: NEPS/nep-12.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+| Title | New paradigm of organization |
+| Author |  [Evan Liu](mailto:evanliuchina@gmail.com), [Xia Wu](https://github.com/xiawu), [Lee Willson](https://github.com/leewillson), [Zhou Xiqiao](https://github.com/zhouxiqiao), [Yong Liu](mailto:liuyong5653@163.com), [Qu Jianwei](https://github.com/i29) |
+| Discussions to | https://github.com/NewORG-Standard-Group/NEPs/issues/1 |
+| Status | Draft |
+| Type | Standard Track |
+| Category | technical |
+| Created | 2020-04-23 |
+| Updated | |
+
+This is the suggested template for NewOrg NRC-X.
```
Comment (evanliuchina) [inline NEPS/nep-12.md:15]: NRC-X => NRC-12 ?

File: NEPS/nep-12.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
+# NEP 12: NewORG Standard (NRC-X)
+
+| Item | Description |
+|:-|:-|
+| NEP | 12 |
+| Title | New paradigm of organization |
+| Author |  [Evan Liu](mailto:evanliuchina@gmail.com), [Xia Wu](https://github.com/xiawu), [Lee Willson](https://github.com/leewillson), [Zhou Xiqiao](https://github.com/zhouxiqiao), [Yong Liu](mailto:liuyong5653@163.com), [Qu Jianwei](https://github.com/i29) |
+| Discussions to | https://github.com/NewORG-Standard-Group/NEPs/issues/1 |
+| Status | Draft |
+| Type | Standard Track |
```
Comment (evanliuchina) [inline NEPS/nep-12.md:10]: plz change Type: Standard Track to Type: NRC

File: NEPS/nep-12.md [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+| Category | technical |
```
Comment (evanliuchina) [inline NEPS/nep-12.md:11]: technical => Technical

File: NEPS/nep-12.md (hunk 4 above)
Comment (arisac) [inline NEPS/nep-12.md]: this line is unnecessary

File: NEPS/nep-12.md (hunk 5 above)
Comment (i29) [reply NEPS/nep-12.md]: done

**Comment by arisac** [comment]
@i29 please change <title> in preamble to "NewORG Standard (NRC-12)"

**Pull request merged**